src/main/java/com/dao/homework/dao/jdbc/ManufacturerJdbcDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+        String query = "SELECT * FROM " + TABLE_NAME + " WHERE " + MANUFACTURER_ID
+                + "=? AND " + DELETED + " = false";
+        Manufacturer manufacturer = new Manufacturer();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, manufacturerId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = getManufacturer(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get data with id:"
+                    + manufacturerId + " from DB ", e);
+        }
+        return Optional.of(manufacturer);
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE " + TABLE_NAME + " SET " + MANUFACTURER_NAME
+                + "= ?, " + MANUFACTURER_COUNTRY + "= ? "
+                + " WHERE " + MANUFACTURER_ID
+                + "= ? AND " + DELETED + " = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.setLong(3, manufacturer.getId());
+            statement.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t update this data ", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long manufacturerId) {
+        String query = "UPDATE " + TABLE_NAME + " SET " + DELETED
+                + "= ?  WHERE " + MANUFACTURER_ID
+                + "= ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setBoolean(1, true);
+            statement.setLong(2, manufacturerId);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t delete data with id:"
+                    + manufacturerId + " from DB ", e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String query = "SELECT * FROM " + TABLE_NAME + " WHERE " + DELETED + " = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            ResultSet resultSet = statement.executeQuery(query);
+            List<Manufacturer> allManufacturers = new ArrayList<>();
+            while (resultSet.next()) {
+                allManufacturers.add(getManufacturer(resultSet));
+            }
+            resultSet.close();
```

Comment:
it is optional to close ResultSet, because when a Statement object is closed, its current ResultSet object, if one exists, is also closed.
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/Statement.html#close()